spec/person_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_relative '../person'
+
+# Describe the Person class and its methods
+RSpec.describe Person do
+  let(:person) { Person.new(25, parent_permission: true, name: 'John Doe') }
+  let(:book) { instance_double('Book') }
+
+  describe '#new' do
+    it 'should be an instance of Person' do
+      expect(person).to be_an_instance_of(Person)
+    end
+
+    it 'should have default attributes' do
+      expect(person.name).to eql('John Doe')
+      expect(person.age).to eql(25)
+      expect(person.parent_permission).to be true
+    end
+  end
```

Comment:
- [ ] You should test the scenario of when the name is not given it should return the default value which is `Unknown` for the name 🙃 The constructor method sets the default value for the optional name parameter if it is not provided.
- [ ] Also the constructor method should return a Person object with provided empty rentals